### DISCUSSION TITLE
Adempiere ERP 3.9.4 in Windows 10. Unable to Install

### DISCUSSION THREAD

**Discussion by khawaraliraan** [adempiere/adempiere#4354] [Q&A]
Dear all

I have been trying to install adempiere ERP 3.9.4 in Windows 10. I have made dozens of attempts. Urfortunately, all times installation was failed.  Screen shots and a log file are attached herewith.  Kindly help and tell me what is the mistake and its solution.

<img width="553" alt="Adempiere Setup page" src="https://github.com/user-attachments/assets/278ac8f0-875f-436f-ad56-cad052282755">

<img width="410" alt="Database and owner user rolll" src="https://github.com/user-attachments/assets/d1a01bf9-47a8-4491-8ccf-78de440337f3">
<img width="375" alt="Enviroment path" src="https://github.com/user-attachments/assets/9bcf8ca9-a0b4-463f-a122-35c56584f2a2">
<img width="422" alt="Enviroment" src="https://github.com/user-attachments/assets/543c0f1e-565b-4d32-b8d3-9bb24879b428">
<img width="369" alt="Tomcat 9 installation" src="https://github.com/user-attachments/assets/f12fa0e3-bf67-4ead-81ef-86c3e8d83a4d">
<img width="675" alt="Failed Notification" src="https://github.com/user-attachments/assets/7db7dc6a-b1d6-47c5-985c-efb6f7637e80">

[2024-10-21_0.log](https://github.com/user-attachments/files/17468973/2024-10-21_0.log)

best regards

**Comment by amirsaifi** [comment]
Try with jetty. It was more easily configured in my case.

**Reply by khawaraliraan** [reply]
Thanks a lot dear Mr. Amir Saifi for support. Your support worked. I have tried with jetty, it worked positively, and build was successful. After installation database import was also successful. GUI is working perfectly in browser at localhost:8888/webui. Server is successfully started. But, Adempiere GUI is not running using RUN_Adempiere in utils folder (attaching screen shot) and client (Adempiere file in lib folder) is also not running and returning error of Java Exception (Screen shot attached). I am trying fix this issue. 
<img width="459" alt="Activity" src="https://github.com/user-attachments/assets/f038343b-19e7-421f-907d-47aaaa447e1b">
<img width="191" alt="Java exception" src="https://github.com/user-attachments/assets/70d2f5ee-4b4b-492e-83a3-cb91b4d0ee49">
